devices.js

```diff
@@ -13469,6 +13469,23 @@ const devices = [
         toZigbee: [],
         exposes: [e.occupancy(), e.battery_low(), e.tamper()],
     },
+    {
+        zigbeeModel: ['HMSZB-110'],
+        model: 'HMSZB-110',
+        vendor: 'Develco',
+        description: 'Temperature & humidity sensor',
+        supports: 'temperature, humidity',
+        fromZigbee: [fz.temperature, fz.humidity],
```

Comment:
Does it also support battery?

- To fromZigbee add `fz.battery`.
- Change `configureKey` to 2
- `await bind(endpoint, coordinatorEndpoint, ['msTemperatureMeasurement', 'msRelativeHumidity']);` -> `await bind(endpoint, coordinatorEndpoint, ['msTemperatureMeasurement', 'msRelativeHumidity', 'genPowerCfg']);`
- below ` await configureReporting.humidity(endpoint);` add `await configureReporting.batteryPercentageRemaining(endpoint);`

Reply:
Yes, it does indeed.
It's powered by 2 AA batteries in case it matters more.

Reply:
Perfect, thanks!